packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -8,7 +8,7 @@ import Map from 'babel-runtime/core-js/map';
 
 import {InvalidEmailAddressError} from './ediscovery-error';
 
-const defaultOptions = {timeoutMs: 30000}; // 30s default timeout
+const defaultOptions = {offset: 0, size: 100, timeoutMs: 30000}; // 30s default timeout
```

Comment:
These should generally be specified in the plugin config so they could be overwritten by users when they instantiate the sdk. I know this was already here. Just mentioning what the current pattern is.